docs/resources/compliance_framework.md

```diff
@@ -0,0 +1,47 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mondoo_compliance_framework Resource - terraform-provider-mondoo"
+subcategory: ""
+description: |-
+  Set Compliance Frameworks for a Mondoo Space.
+---
+
+# mondoo_compliance_framework (Resource)
+
+Set Compliance Frameworks for a Mondoo Space.
+
+## Example Usage
+
+```terraform
+provider "mondoo" {
+  region = "us"
+}
+
+variable "mondoo_org" {
+  description = "The Mondoo Organization ID"
+  type        = string
+  default     = "my-org-1234567"
+}
+
+# Create a new space
+resource "mondoo_space" "my_space" {
+  name   = "Framework Space"
+  org_id = var.mondoo_org
+}
+
+resource "mondoo_compliance_framework" "compliance_framework_example" {
```

Comment:
We should use `mondoo_framework_assignment` as defined in https://github.com/mondoohq/terraform-provider-mondoo/issues/103